components/script/dom/htmliframeelement.rs

```diff
@@ -331,7 +331,9 @@ impl HTMLIFrameElement {
         self.pipeline_id.get()
             .and_then(|pipeline_id| ScriptThread::find_document(pipeline_id))
             .and_then(|document| {
-                if self.global().get_url().origin() == document.global().get_url().origin() {
+                let contained_url = document.global().get_url();
+                if self.global().get_url().origin() == contained_url.origin() ||
+                   contained_url.as_str() == "about:blank" {
```

Comment:
Could you check if this change affects the various Range mutation tests that were disabled?